acceptance/bundle/telemetry/deploy-no-uuid/output.txt

```diff
@@ -35,7 +35,14 @@ Deployment complete!
         "resource_app_count": 0,
         "resource_job_ids": [
           "1"
-        ]
+        ],
+        "experimental": {
+          "configuration_file_count": 0,
```

Comment:
`configuration_file_count` and `target_count` are incorrectly serialized to 0 right now.
1. `configuration_file_count` addressed in https://github.com/databricks/cli/pull/2589
2. `target_count` fix is in progress